src/reason_pprint_ast.re

```diff
@@ -0,0 +1,6694 @@
+        };
+      isSequencey
+    };
+    let isSequencey = convertIsListyToIsSequencey isListy;
+    let inline preSpace::preSpace=false postSpace::postSpace=false labelTerm term =>
+      makeList
+        inline::(true, true)
+        postSpace::postSpace
+        preSpace::preSpace
+        indent::0
+        break::Never
+        [labelTerm, term];
+    let breakline labelTerm term =>
+      makeList inline::(true, true) indent::0 break::Always_rec [labelTerm, term];
+    let insertBlankLines n term =>
+      if (n == 0) {
+        term
+      } else {
+        makeList
+          inline::(true, true)
+          indent::0
+          break::Always_rec
+          (Array.to_list (Array.make n (atom "")) @ [term])
+      };
+    let string_after s n => String.sub s n (String.length s - n);
+    let wrapComment txt => "/*" ^ txt ^ "*/";
+    let formatComment_ txt => {
+      let commLines = Re_str.split_delim (Re_str.regexp "\n") (wrapComment txt);
+      switch commLines {
+      | [] => atom ""
+      | [hd] =>
+        makeList
+          inline::(true, true) postSpace::true preSpace::true indent::0 break::IfNeed [atom hd]
+      | [zero, one, ...tl] =>
+        let hasMeaningfulContentOnLineZero = lineZeroHasMeaningfulContent zero;
+        let attemptRemoveCount = smallestLeadingSpaces [one, ...tl];
+        let leftPad =
+          if (beginsWithStar one) {
+            1
+          } else if hasMeaningfulContentOnLineZero {
+            spaceBeforeMeaningfulContent zero
+          } else {
+            1
+          };
+        let padNonOpeningLine s => {
+          let numLeadingSpaceForThisLine = numLeadingSpace s;
+          if (String.length s === 0) {
+            ""
+          } else {
+            String.make leftPad ' ' ^
+            string_after s (min attemptRemoveCount numLeadingSpaceForThisLine)
+          }
+        };
+        let lines = [zero, ...List.map padNonOpeningLine [one, ...tl]];
+        makeList inline::(true, true) indent::0 break::Always_rec (List.map atom lines)
+      }
+    };
+    let formatComment locOpt::locOpt=? txt => {
+      let list = formatComment_ txt;
+      switch locOpt {
+      | None => list
+      | Some loc => SourceMap loc list [@implicit_arity]
+      }
+    };
+
+    /** [hasComment layout] checks if a layout has comment attached to it */
+    let rec hasComment =
+      fun
+      | WithEOLComment _ _ [@implicit_arity] => true
+      | SourceMap _ sub [@implicit_arity] => hasComment sub
+      | _ => false;
+    let rec append space::space=false txt =>
+      fun
+      | SourceMap loc sub [@implicit_arity] =>
+        SourceMap loc (append space::space txt sub) [@implicit_arity]
+      | Sequence config l [@implicit_arity] when snd config.wrap != "" => {
+          let sep = if space {" "} else {""};
+          Sequence {...config, wrap: (fst config.wrap, snd config.wrap ^ sep ^ txt)} l
+          [@implicit_arity]
+        }
+      | Sequence conf
```

Comment:
I think when converting you should get rid of these. You do that by passing `-assume-explicit-arity`, and then fixing any compiler errors by manually fixing constructors in the `.re` file.